src/server.js

```diff
@@ -14,6 +14,8 @@ import manageConnections from './ws-connections'
 import createDB from './db'
 import configCleanup from './cleanup'
 import apiKeyAuth from './middleware/api-key-auth'
+import healthCheck from './healthcheck'
+import checkDatabase from './middleware/healthcheck-middleware'
```

Comment:
I would name the file rather `response-guard` than `healthcheck-middleware` and `checkDatabase` to `responseGuard`. There does not happen any database checking actually, so it can be misleading